bundle/credentials.go

```diff
@@ -0,0 +1,8 @@
+package bundle
+
+// Credential represents the definition of a CNAB credential
+type Credential struct {
+	Location    `json:",squash" mapstructure:",squash"`
```

Comment:
Is the `squash` annotation available for `json` ? 🤔

Reply:
Oops sorry, that's not needed :(